com.sublimetext.three.yaml

```diff
@@ -62,6 +62,7 @@ modules:
           - PCFILE="Package Control.sublime-package"
           - PCLOCATION="$HOME/.var/app/com.sublimetext.three/config/sublime-text-3/Installed Packages"
           - if ! test -f "$PCLOCATION/$PCFILE"; then
+                mkdir -p "$PCLOCATION"
```

Comment:
Probably for consistency this should either have a dash, or the other lines should have their dashes removed. (I did not know about [YAML Plain Flow Scalars](https://yaml-multiline.info/) before now!)

Reply:
Fixed.